compiler/damlc/lib/DA/Cli/Visual.hs

```diff
@@ -143,9 +143,12 @@ graphEdges lookupData tplChcActions = map (\(chn1, chn2) -> (nodeIdForChoice loo
 subGraphHeader :: LF.Template -> String
 subGraphHeader tpl = "subgraph cluster_" ++ (DAP.renderPretty $ head (LF.unTypeConName $ LF.tplTypeCon tpl)) ++ "{\n"
 
+choiceStyleColorCode :: Bool -> String
+choiceStyleColorCode True = "red"
+choiceStyleColorCode False = "green"
```

Comment:
This is a little strange. Is `Bool` supposed to be `IsConsuming`? You can also use a new data type `data Consumption = Consuming | NonConsuming` to avoid this kind of thing.

Reply:
Thanks that should have been `IsConsuming`